pkg/controller/controller.go

```diff
@@ -470,6 +470,17 @@ func getClass(ingressType string) string {
 	return "nginx"
 }
 
+func getIssuerKind(issuerType string) string {
```

Comment:
Please add a basic test for this which runs through the three permutations:

value = "ClusterIssuer", "Issuer", or ""

Reply:
Done